src/ImageSequencer.js

```diff
@@ -421,6 +306,8 @@ ImageSequencer = function ImageSequencer(options) {
       this.sequences = require('./SavedSequences.json');
   }
 
+  var str = require('./Strings.js')(this.steps, this.images, modulesInfo, addSteps, copy)
```

Comment:
I think the issue is that `loadNewModule()` is being run out of this scope. I think you may need to assign `this` to a variable in this scope, so it doesn't get overwritten in another scope. See:

`var this_` at top - that's creating a closure, and so a variable that will persist in the scope where `loadNewModule()` is defined, no matter where you run it. 

I think 2 things -- `this_` is poorly named... could it be renamed `sequencer`? Is that appropriate?

2nd, could we then refer to `sequencer.images` in `loadNewModule()` so that we are sure to get the correct reference? Likewise, I think it's a good idea to pass the `sequencer.images` value in by reference when we run `Strings.js`.